<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgPaths</class>
 <widget class="QDialog" name="CQDlgPaths">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Path Properties</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Visual properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqShowOrientation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>251</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Show orientation of points</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqShowPathLine">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>25</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Show path line</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqShowPosition">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>251</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Show current position on path</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Line size [pt]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqLineSize">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>65</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>65</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Size of ctrl points [m]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqControlPointSize">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>65</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="qqAdjustColor">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>45</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Adjust line color</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="qqDistanceCombo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>281</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="qqLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Distance unit</string>
   </property>
  </widget>
  <widget class="QToolButton" name="qqShowShapingDialog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Show path shaping dialog</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="qqCopyToClipboard">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Copy path data to status bar</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="qqCreateEquivalentObject">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Create similar object</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqDistanceCombo</tabstop>
  <tabstop>qqShowShapingDialog</tabstop>
  <tabstop>qqShowOrientation</tabstop>
  <tabstop>qqShowPathLine</tabstop>
  <tabstop>qqShowPosition</tabstop>
  <tabstop>qqAdjustColor</tabstop>
  <tabstop>qqLineSize</tabstop>
  <tabstop>qqControlPointSize</tabstop>
  <tabstop>qqCreateEquivalentObject</tabstop>
  <tabstop>qqCopyToClipboard</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
